htdocs/core/lib/functionsnumtoword.lib.php

```diff
@@ -247,6 +247,7 @@ function dolNumberToWord($numero, $langs, $numorcurrency = 'number')
 		}
 		return $entexto;
 	}
+	return -1;
```

Comment:
If you return -1 here (like it is done for some previous error check), you must also modify the comment doc of the method by changing 
`* @return string Text of the number`
into
`* @return int|string Text of the number`